overrides/ios-override.json

```diff
@@ -189,6 +189,21 @@
                 ]
             }
         },
+        "duckPlayerNative": {
+            "state": "enabled",
+            "exceptions": [],
+            "settings": {
```

Comment:
Do you need this in both features/ and the override? Especially with the schema if it was hooked in, I'd drop it out of one of them.

Reply:
I hadn't realised that was safe to do. This is great! Fixed.